{"primaryContentSections":[{"declarations":[{"platforms":["Mac Catalyst"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unknown"}]}],"kind":"declarations"}],"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"unknown","kind":"identifier"}],"externalID":"s:19AdyenAuthentication0aB5ErrorO7unknownyA2CmF","roleHeading":"Case","symbolKind":"case","role":"symbol","title":"AdyenAuthenticationError.unknown","modules":[{"name":"AdyenAuthentication"}]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/adyenauthentication\/adyenauthenticationerror\/unknown"]}],"kind":"symbol","sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication\/AdyenAuthenticationError\/unknown"},"hierarchy":{"paths":[["doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication","doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication\/AdyenAuthenticationError"]]},"abstract":[{"type":"text","text":"Unknown error occured."}],"schemaVersion":{"minor":3,"patch":0,"major":0},"references":{"doc://com.adyen.AdyenAuthentication/documentation/AdyenAuthentication":{"type":"topic","url":"\/documentation\/adyenauthentication","title":"AdyenAuthentication","abstract":[{"type":"text","text":"AdyenAuthentication SDK Provides reusable and easy to use two factor authentication"},{"type":"text","text":" "},{"type":"text","text":"for security sensitive use cases like banking, issuing and PSD2 strong customer authentication."}],"identifier":"doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication","kind":"symbol","role":"collection"},"doc://com.adyen.AdyenAuthentication/documentation/AdyenAuthentication/AdyenAuthenticationError":{"url":"\/documentation\/adyenauthentication\/adyenauthenticationerror","title":"AdyenAuthenticationError","role":"symbol","kind":"symbol","navigatorTitle":[{"text":"AdyenAuthenticationError","kind":"identifier"}],"identifier":"doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication\/AdyenAuthenticationError","abstract":[{"text":"An Error thrown by ","type":"text"},{"type":"codeVoice","code":"AuthenticationService"},{"text":".","type":"text"}],"type":"topic","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"AdyenAuthenticationError","kind":"identifier"}]},"doc://com.adyen.AdyenAuthentication/documentation/AdyenAuthentication/AdyenAuthenticationError/unknown":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unknown"}],"role":"symbol","abstract":[{"type":"text","text":"Unknown error occured."}],"identifier":"doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication\/AdyenAuthenticationError\/unknown","kind":"symbol","type":"topic","title":"AdyenAuthenticationError.unknown","url":"\/documentation\/adyenauthentication\/adyenauthenticationerror\/unknown"}}}